common/utils/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -106,6 +106,20 @@ object SparkException {
       messageParameters = Map("message" -> msg),
       cause = cause)
   }
+
+  /**
+   * This is like the Scala require precondition, except it uses SparkException.
+   * @param requirement The requirement you want to check
+   * @param errorClass The error class to type if the requirement isn't passed
+   * @param messageParameters Message parameters to append to the message
+   */
+  def checkArgs(requirement: Boolean,
+                errorClass: String,
+                messageParameters: Map[String, String]): Unit = {
```

Comment:
```suggestion
  def checkArgs(
      requirement: Boolean,
      errorClass: String,
      messageParameters: Map[String, String]): Unit = {
```

Reply:
Thanks! My IntelliJ prefers the first. Do you have specific settings set up to make it default to yours?